go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -4,7 +4,6 @@
 require "dependabot/errors"
 require "dependabot/logger"
 require "dependabot/go_modules/file_updater"
-require "dependabot/go_modules/native_helpers"
```

Comment:
I suspect this is an unused import... probably never cleaned up when we previously thinned out other commands from the native helpers...